apps/electron/src/main.js

```diff
@@ -109,8 +109,13 @@ function askToQuit() {
  * Allows processes to escalate errors to be shown in electron
  * @param {string} error
  */
-function escalateError(error) {
-  dialog.showErrorBox('An unrecoverable error occurred', error);
+function escalateError(error, unrecoverable = true) {
+  if (unrecoverable) {
+    dialog.showErrorBox('An unrecoverable error occurred', error);
+    //TODO: should we quit here
```

Comment:
I dont have a strong opinion.
If I had to throw an opinion dice I would say: "yes if it is not too much work"